libs/markets/src/lib/components/market-info/tooltip-mapping.tsx

```diff
@@ -106,4 +106,43 @@ export const tooltipMapping: Record<string, ReactNode> = {
   ),
+  commitmentMinimumTimeFraction: t(
+    `This is a market parameter. It specifies the minimum fraction of time LPs must spend 'on the book' providing their committed liquidity.`
+  ),
+  feeCalculationTimeStep: t(
+    'This is a market parameter. This is providers fee calculation time step. Specifies how often the quality of liquidity supplied by the LPs is evaluated and fees arising from that period are earmarked for specific parties.'
+  ),
+  performanceHysteresisEpochs: t(
+    'This is a market parameter. Specifies the number of epochs over which past performance will continue to affect rewards.'
+  ),
+  SLACompetitionFactor: t(
+    `This is a market parameter. The SLA completion factor. Specifies the maximum fraction of their accrued fees an LP that meets the SLA implied by market.liquidity.commitmentMinTimeFraction will lose to liquidity providers that achieved a higher SLA performance than them. `
+  ),
+  bondPenaltyParameter: t(
+    'This is a network parameter. It is used to calculate the penalty to liquidity providers when they cannot support their open position with the assets in their margin and general accounts. Valid values: any decimal number >= 0 with a default value of 0.1.'
+  ),
+  nonPerformanceBondPenaltySlope: t(
+    'This is a network parameter. It is used to calculate by how much the LP bond is slashed if an LP fails to reach the minimum SLA. Valid values: any decimal number >= 0 with a default value of 2.0.'
+  ),
+  nonPerformanceBondPenaltyMax: t(
+    'This is a network parameter. It is used to calculate how much is the LP bond slashed if they fail to reach the minimum SLA. Valid values: any decimal number >= 0 and <=1.0 with a default value of 0.5.'
+  ),
+  maximumLiquidityFeeFactorLevel: t(
+    'This is a network parameter. It is used to validate the proposed fee amounts that are submitted as part of the LP commitment transaction. Note that a value of 0.05 = 5%. Valid values are: any decimal number >=0 and <=1. Default value 1.'
+  ),
+  stakeToCCYVolume: t(
+    `This is a network parameter. It is used to translate an LP's commitment amount to an obligation. Any decimal number >0 with default value 1.0.`
+  ),
+  epochLength: t(
+    'This is a network parameter. LP rewards from liquidity fees are paid out once per epoch. How much they receive depends on whether they met the liquidity SLA and their previous performance in recent epochs (defined by market.liquidity.performanceHysteresisEpochs).'
+  ),
+  earlyExitPenalty: t(
+    'This is a network parameter. It sets how much an LP forfeits of their bond if they reduce their commitment while the market is below target stake. If set to 0 there is no penalty for early exit, if set to 1 their entire bond is forfeited if they exit their entire commitment, if set >1, their entire bond will be forfeited for exiting 1/earlyExitPenalty of their commitment amount. Must be a decimal ≥0. '
```

Comment:
it is neither. `earlyExitPenalty` is left unformatted and a whole value

Reply:
Could it be displayed as a percentage, since 1 = 100%? Otherwise I'll reword to say it's a factor, but showing it as a % would make it easier to understand.